Apps/W1/Shopify/app/src/Payments/Enums/ShpfyDisputeStatus.Enum.al

```diff
@@ -0,0 +1,35 @@
+namespace Microsoft.Integration.Shopify;
+enum 30150 "Shpfy Dispute Status"
+{
+    Caption = 'Shopify Dispute Status';
+    Extensible = true;
```

Comment:
I don't see any reason why this enum should be extensible either. 
The documentation here could be improved by adding a summary tag to each value with the describing text from the shopify documentation. https://shopify.dev/docs/api/admin-rest/2024-01/resources/dispute#resource-object

Reply:
Updated.